submatrix/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:latest as submatrix
+
+RUN apk add synapse envsubst py3-curl
+
+ARG MATRIX_HOSTNAME=localhost
```

Comment:
can move this down to right before the envsubst so that changing it trashes the cache of fewer layers